<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>208</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="configFileLabel">
       <property name="text">
        <string>配置文件:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dataDirectoryLabel">
       <property name="text">
        <string>数据目录:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="odBrowseButton">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="outputDirectoryEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="projectDirectoryLabel">
       <property name="text">
        <string>母工程目录:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="cfBrowseButton">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="outputDirectoryLabel">
       <property name="text">
        <string>输出目录:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="projectDirectoryEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="rdBrowseButton">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="resourceDirectoryEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="resourceDirectoryLabel">
       <property name="text">
        <string>资源目录:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="dataDirectoryEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="ddBrowseButton">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pdBrowseButton">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="configFileEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="stepFileLabel">
       <property name="text">
        <string>步骤文件:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="stepFileEdit"/>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="sfBrowseButton">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumain">
    <property name="title">
     <string>main</string>
    </property>
    <addaction name="actionMain"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menumain"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMain">
   <property name="text">
    <string>Main</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
